test/records.test.mjs

```diff
@@ -147,4 +159,37 @@ describe('aos Records', async () => {
     assert(record.transactionId === ''.padEnd(43, '3'));
     assert(record.ttlSeconds === 3600);
   });
+
+  it('should set valid undernames from length 1 to 61 and remove them after', async () => {
```

Comment:
This seems excessive given the coverage we have at the unit test level.

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/60/commits/e410082fc1fb32ea61d18f2e71b9bf5f12c4cc4e